### DISCUSSION TITLE
急 SCP-096突破收容

### DISCUSSION THREAD

**Discussion by xiaosheng31** [share121/inter-knot#3016] [其他]
收容物096于今日突破收容据卫星侦测096一路向北来到新艾利都市进入空洞至今下落不明负责收容的九尾狐小队因没有进入空洞的技术
现招募一名绳匠指引小队进入空洞当中下面说明注意事项

1. 收容物名字SCP-096，又名“羞涩的人”，[SCP - 收容失效]的高度危险人形SCP.

2.描述:SCP-096是一个通常极其温顺的人型生物。但是，当有人看到SCP-096的脸时，无论是直接看到，还是看到了视频，甚至是看到了照片，它都会进入严重的悲伤状态。SCP-096会用手蒙住脸，开始尖叫、哭泣并发出含混不清的声音，然后追向看到它脸的人.

![screenshot-1724040949179](https://github.com/user-attachments/assets/135aa52d-da00-4e07-843e-60af24109a09)
最后:不要看他的脸

**Comment by lllliufeng** [comment]
这个委托，不可言，不可视，不可闻，不可接

**Reply by xiaosheng31** [reply]
报酬丰厚

**Reply by chenx198** [reply]
造反D请求出战！

**Reply by Doctor-Bright** [reply]
> 造反D请求出战！

传奇电炮dd是吧

**Comment by GrunderIndustries** [comment]
您可以尝试使用我司的MPBM产品对其进行无效化处理

**Reply by xiaosheng31** [reply]
二十七次。在过去的6个月里，一共进行二十七次针对SCP-096的处决。所有处决均由当初让它突破收容的Daniels博士指导进行。

Daniels在过去六个月里在牢房里度过。每当处决日他会被护送到测试室监督处决，再返回那个牢房。他的处决决定被推迟以便他能监督SCP-096的处决，但O5在失去他们的耐心。Daniels博士是一位被判刑的人员，如果他所有的尝试失败，这个处决项目的负责人将会被别人取代。

他只有一个月的时间。无论结果如何，在这个月末他将会被处决但是scp-096此时突破收容处决计划被迫终止。
此前处决最新发现scp-096身体硬度比最新发现的物质碳块还要坚硬并且免疫一切伤害.
若此次回收成功将启动最终计划:将scp-096发射至太空并将其送入黑洞当中。

**Comment by Pheathon** [comment]
这个委托我接下了

**Reply by xiaosheng31** [reply]
此话当真

**Comment by 1ctrl-cv** [comment]
特遣队Iota-10的队员告诉你：可以尝试低着头往它头上套不透明的袋子来回收它

**Reply by xiaosheng31** [reply]
已备注

**Comment by 1492830678** [comment]
这个委托我接...算了算了<img src="https://bbs-static.miyoushe.com/static/2024/06/19/f7d6b0099c022d557d12c6662b132406_7125757908673767353.png" alt="流萤 也挺好" width="75" >

**Reply by xiaosheng31** [reply]
各位绳匠可根据自己的能力事实而定

**Comment by opqphp** [comment]
<img src="https://upload-bbs.mihoyo.com/upload/2022/11/14/829f0a1d533765b80769dd1a38dc338c_9182108590963360703.png" alt="阿姬惊讶" width="75" >这个委托我怕了

**Comment by Soldier11-ObsidianBarracks** [comment]
防卫军准备发射最新研发的地对地导弹CRV89/E，请告诉我们具体坐标，谢谢

**Comment by Larksonguesinaspic** [comment]
听说上头领导要介入了（我是市政的），嘿嘿有好戏看咯<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" ><img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >
